spec/requests/api/v1/allocations_controller_create_spec.rb

```diff
@@ -7,7 +7,10 @@
 
   let(:response_json) { JSON.parse(response.body) }
   let(:resource_to_json) do
-    JSON.parse(ActionController::Base.render(json: allocation, include: AllocationSerializer::INCLUDED_ATTRIBUTES))
+    resource = JSON.parse(ActionController::Base.render(json: allocation, include: AllocationSerializer::INCLUDED_ATTRIBUTES))
+    resource['data']['relationships']['moves']['data'] = UnorderedArray(*resource.dig('data', 'relationships', 'moves', 'data'))
```

Comment:
pretty ugly but unfortunately comparing nested arrays in JSON is failing due to ordering in the array itself, would appreciate thoughts on a better way in doing this